packages/office-ui-fabric-react/src/components/Callout/CalloutContent.tsx

```diff
@@ -79,7 +79,8 @@ export class CalloutContent extends BaseComponent<ICalloutProps, ICalloutState>
   }
 
   public componentWillUpdate(newProps: ICalloutProps) {
-    if (newProps.targetElement !== this.props.targetElement || newProps.target !== this.props.target) {
+    // If the target element changed, find the new one. If we are tracking target with class name, always find element because we do not know if fabric has rendered a new element and disposed the old element.
+    if (newProps.targetElement !== this.props.targetElement || newProps.target !== this.props.target || typeof(newProps.target) === 'string' || newProps.target instanceof String) {
```

Comment:
isn't the `newProps.target instanceof String` redundant with `typeof newProps.target === 'string'` ?